src/util.cpp

```diff
@@ -584,7 +584,9 @@ void ArgsManager::ForceSetArg(const std::string& strArg, const std::string& strV
     mapMultiArgs[strArg] = {strValue};
 }
 
-
+bool HelpRequested(const ArgsManager& args) {
+  return args.IsArgSet("-?") || args.IsArgSet("-h") || args.IsArgSet("-help");
```

Comment:
Indentation is off (should use 4 spaces)

Reply:
Note: Don't forget to install `clang-format` and run the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script to preempt whitespace nitpicking.

Reply:
Fixed, and thanks - I need to set up a hook for that. :P